File: chapter_1/localVariables.js

function a() {
	let spam = "Ant";
	document.write("a: Spam is " + spam + "<br />");
	b();
	document.write("a: Spam is " + spam + "<br />");
}


function b() {
	let spam = "Bobcat";
	document.write("b: Spam is " + spam + "<br />");
	c();
	document.write("b: Spam is " + spam + "<br />");
}

function c() {
	let spam = "Coyote";
	document.write("c: Spam is " + spam + "<br />");
}

a()

File: chapter_1/localVariables.py
def a():
    spam = "Ant"
    print("a: Spam is " + spam)
    b()
    print("a: Spam is " + spam)


def b():
    spam = "Bobcat"
    print("b: Spam is " + spam)
    c()
    print("b: Spam is " + spam)

def c():
    spam = "Coyote"
    print("c: Spam is " + spam)


if __name__ == "__main__":
    a()
